samples/webview/webview.cpp

```diff
@@ -309,31 +304,32 @@ WebFrame::WebFrame(const wxString& url) :
     EnableFullScreenView(); // Enable native fullscreen API on macOS
 
     wxBoxSizer* topsizer = new wxBoxSizer(wxVERTICAL);
+    long toolbarStyle = wxTB_TEXT;
 
     // Create the toolbar
-    m_toolbar = CreateToolBar(wxTB_TEXT);
-    m_toolbar->SetToolBitmapSize(wxSize(32, 32));
-
-    wxBitmap back = wxArtProvider::GetBitmap(wxART_GO_BACK , wxART_TOOLBAR);
-    wxBitmap forward = wxArtProvider::GetBitmap(wxART_GO_FORWARD , wxART_TOOLBAR);
-    #ifdef __WXGTK__
-        wxBitmap stop = wxArtProvider::GetBitmap("gtk-stop", wxART_TOOLBAR);
-    #else
-        wxBitmap stop = wxBitmap(stop_xpm);
-    #endif
-    #ifdef __WXGTK__
-        wxBitmap refresh = wxArtProvider::GetBitmap("gtk-refresh", wxART_TOOLBAR);
-    #else
-        wxBitmap refresh = wxBitmap(refresh_xpm);
-    #endif
+#ifndef wxHAS_SVG
+    toolbarStyle |= wxTB_NOICONS;
+#endif
+    m_toolbar = CreateToolBar(toolbarStyle);
+
+    wxSize sizeBitmap = wxArtProvider::GetNativeSizeHint(wxART_TOOLBAR);
+
+    if ( !sizeBitmap.IsFullySpecified() )
+        sizeBitmap.Set(32, 32);
```

Comment:
Just curious, why was 32 chosen in the original code (regardless of the platform and DPI)?

Reply:
I don't think there is any deep reason for this, it's just a reasonable default. For most platforms `GetNativeSizeHint()` should return valid values, I think.